frigate/video.py

```diff
@@ -781,10 +790,16 @@ def process_frames(
             logger.info(f"{camera_name}: frame {frame_time} is not in memory store.")
             continue
 
-        # look for motion if enabled
+        # look for motion if enabled and ptz is not moving
+        # ptz_moving_at_frame_time() always returns False for
+        # non ptz/autotracking cameras
         motion_boxes = (
             motion_detector.detect(frame)
-            if motion_enabled.value and ptz_stopped.is_set()
+            if motion_enabled.value
+            and ptz_stopped.is_set()
+            and not ptz_moving_at_frame_time(
+                frame_time, ptz_start_time.value, ptz_stop_time.value
+            )
```

Comment:
Maybe this could be made simpler with a single function that takes these values and is called `is_ptz_moving` or something

Reply:
I thought about that, but the only place where I check for `ptz_stopped.is_set()` and need to check the `frame_time` at the same time is there. And thinking about it again, it might not strictly be even necessary. Let me do a little more testing and see if I can remove line 799 altogether.